src/execution/StepExecutionManager.ts

```diff
@@ -178,17 +251,42 @@ export class StepExecutionManager {
           transaction = await signer.sendTransaction(transactionRequest)
 
           // STEP 4: Wait for the transaction
-          process = statusManager.updateProcess(step, process.type, 'PENDING', {
-            txHash: transaction.hash,
-            txLink:
-              fromChain.metamask.blockExplorerUrls[0] +
-              'tx/' +
-              transaction.hash,
-          })
+          if (isMultisigSigner) {
+            statusManager.updateExecution(step, 'ACTION_REQUIRED')
```

Comment:
We need to double-check if we change execution to ACTION_REQUIRED somewhere because maybe it's not needed.

Reply:
Please remove as per discussion